cmd/export_assets.go

```diff
@@ -16,14 +16,22 @@ var assetsCmd = &cobra.Command{
 	Long:  `Exports the assets that are created from payment operations over a specified ledger range`,
 	Run: func(cmd *cobra.Command, args []string) {
 		cmdLogger.SetLevel(logrus.InfoLevel)
-		endNum, strictExport, isTest, isFuture, extra := utils.MustCommonFlags(cmd.Flags(), cmdLogger)
+		endNum, strictExport, isTest, isFuture, extra, useCaptiveCore, datastoreUrl := utils.MustCommonFlags(cmd.Flags(), cmdLogger)
 		cmdLogger.StrictExport = strictExport
 		startNum, path, limit := utils.MustArchiveFlags(cmd.Flags(), cmdLogger)
 		cloudStorageBucket, cloudCredentials, cloudProvider := utils.MustCloudStorageFlags(cmd.Flags(), cmdLogger)
+		env := utils.GetEnvironmentDetails(isTest, isFuture, datastoreUrl)
 
 		outFile := mustOutFile(path)
 
-		paymentOps, err := input.GetPaymentOperations(startNum, endNum, limit, isTest, isFuture)
+		var paymentOps []input.AssetTransformInput
+		var err error
+
+		if useCaptiveCore {
+			paymentOps, err = input.GetPaymentOperationsHistoryArchive(startNum, endNum, limit, isTest, isFuture)
+		} else {
+			paymentOps, err = input.GetPaymentOperations(startNum, endNum, limit, env, useCaptiveCore)
```

Comment:
I like your idea of eventually refactoring this History Archives method to use lcm directly. I think it's cleaner and less confusing